docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -52,22 +52,31 @@ NOTE: In case the Intel SGX APT source repository is not added to your system. R
 
 On Ubuntu 18.04:
 ```bash
-echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intelsgx.list
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+```
+
+On Ubuntu 20.04:
+```bash
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
 ```
 
 Add the key to the list of trusted keys used by the apt to authenticate packages:
 ```bash
 wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
 ```
 
-Update the apt
+Update apt package database
 ```bash
 sudo apt-get update
 ```
 
 - Or Install Intel DCAP Quote Provider with dpkg manually
```

Comment:
Bullet pointing this seems awkward as it is indented compared to the paragraph that follows it